Assets/Scripts/Controllers/BuildingPlacement.cs

```diff
@@ -43,6 +44,13 @@ private void Start()
 
         private void Update()
         {
+#if UNITY_EDITOR
+            //Random debug code
+            if (Input.GetKeyDown(KeyCode.F10))
```

Comment:
What is the purpose of `SetFirstCard(0, testBuilding);`?

Reply:
This is for testing buildings. It got very annoying having to restart the game over and over hoping that the building I wanted to place would show up.